src/Runner.Worker/Container/ContainerHooks/ContainerHookManager.cs

```diff
@@ -215,11 +203,37 @@ private string GenerateResponsePath()
             return Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Temp), ResponseFolderName, $"{Guid.NewGuid()}.json");
         }
 
-        private static string GetPrependPath(IExecutionContext context)
+        private string GetPrependPath(IExecutionContext context)
         {
             return string.Join(Path.PathSeparator.ToString(), context.Global.PrependPath.Reverse<string>());;
         }
 
+        private HookResponse GetResponse(HookInput input)
+        {
+            HookResponse response = null;
+
+            if (string.IsNullOrEmpty(input.ResponseFile) || !File.Exists(input.ResponseFile))
+            {
+                Trace.Info($"Response file for the hook script at '{HookIndexPath}' running command '{input.Command}' not found.");
+                if (input.Command == HookCommand.PrepareJob)
+                {
+                    throw new Exception($"Response file for the hook script at '{HookIndexPath}' running command '{input.Command}' not found.");
```

Comment:
`$"Response file is required but not found for the hook script at '{HookIndexPath}' running command '{input.Command}'"`